Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/modify.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (C) 2025 Apple Inc. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+# 1.  Redistributions of source code must retain the above copyright
+#     notice, this list of conditions and the following disclaimer.
+# 2.  Redistributions in binary form must reproduce the above copyright
+#     notice, this list of conditions and the following disclaimer in the
+#     documentation and/or other materials provided with the distribution.
+#
+# THIS SOFTWARE IS PROVIDED BY APPLE INC. AND ITS CONTRIBUTORS "AS IS" AND
+# ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL APPLE INC. OR ITS CONTRIBUTORS BE LIABLE FOR
+# ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+# CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+# OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import os
+import re
+import sys
+
+from .command import Command
+from .trace import Trace, Relationship
+from .commit import Commit as CommitProgram
+
+from webkitbugspy import Tracker, bugzilla, radar
+from webkitcorepy import arguments, run, string_utils, Terminal
+from webkitscmpy import local, log
+from ..commit import Commit
+
+
+class Modify(Command):
+    name = 'modify'
+    help = 'Perform a given modification to an existing bug,' \
+           'passing provided info to the associated radar.'
+
+    @classmethod
+    def parser(cls, parser, loggers=None):
+        parser.add_argument(
+            'arguments', nargs='+',
+            type=str,
+            help='String representation of the commit hash(es) to modify',
+        )
+        parser.add_argument(
+            '--keyword', '-k',
+            dest='keyword',
+            required=True,
+            help='Keyword to add to radar from commit hash(es)',
+            type=str,
+        )
+
+    @classmethod
+    def main(cls, args, repository, **kwargs):
+        keyword = args.keyword
+        if not keyword:
+            print("No keyword provided. Exiting.")
+            return 1
+
+        rdar = None
+
+        for tracker in Tracker._trackers:
+            if isinstance(tracker, radar.Tracker):
+                rdar = tracker
+                break
+        if not rdar or not rdar.radarclient():
+            if not rdar:
+                sys.stderr.write('Radar not available. Please validate credentials. \n')
+            return 255
+
+        for commit_hash in args.arguments:
```

Comment:
This looks really similar to `git-webkit find`. Does that command have something that returns a list of matching commits, maybe? Another question would be if you could make this "pipeable", ie, make something like this work:
`git log --online -20 | git-webkit modify ...`

Reply:
As a side note, "integrate with `git-webkit find`" and "pipeable" are mutually exclusive.